bin/kubectl-crossplane-stack-build

```diff
@@ -1,10 +1,23 @@
 #!/usr/bin/env bash
 
 function usage {
-  echo "Usage: kubectl crossplane stack [-h|--help] build [ARGS ...]" >&2
+  echo "Usage: kubectl crossplane stack [-h|--help] build [STACK_IMAGE_NAME] [ARGS ...]" >&2
   echo "" >&2
-  echo "ARGS: An alternate recipe to build, or other arguments to pass through" >&2
-  echo 'to `make`.' >&2
+  echo "" >&2
+  echo "STACK_IMAGE_NAME is the name of the stack in the local docker image" >&2
+  echo "registry to publish. It will be passed to docker, so it can start with" >&2
```

Comment:
Nitpick: it is passed to docker eventually in the stacks we maintain, but it is not passed directly to docker by the CLI tool

Reply:
This means to imply that the argument will be "passed to docker" the CLI tool, not the Docker Hub registry.  FWIW, I copied this text from a venerable source: https://github.com/crossplane/crossplane-cli/blame/master/bin/kubectl-crossplane-stack-publish#L8-L11

Reply:
I can snaz up the text and see about adding more examples.